x-pack/test_serverless/shared/config.base.ts

```diff
@@ -60,6 +60,13 @@ export default async () => {
             appenders: ['deprecation'],
           },
         ])}`,
+        // This ensures that we register the Security SAML API endpoints.
+        // In the real world the SAML config is injected by control plane.
+        '--xpack.cloud.id=ftr_fake_cloud_id',
+        `--xpack.security.authc.providers=${JSON.stringify({
+          basic: { 'cloud-basic': { order: 0 } },
```

Comment:
__question:__ I think we won't have `basic` in Serverless, is there any reason we still need this in the config?

Reply:
Good catch. I accidentally left this in when I copied the config over.